src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Appearance/WindowBackdrop.cs

```diff
@@ -50,11 +50,23 @@ internal static bool ApplyBackdrop(System.Windows.Window window, WindowBackdropT
             return ApplyBackdropCore(window, backdropType);
         }
 
-        window.Loaded += (sender, _) =>
+        RoutedEventHandler loadedHandler = null;
+        loadedHandler = (sender, _) =>
         {
+            IntPtr windowHandle =
+                new WindowInteropHelper(sender as System.Windows.Window)?.Handle ?? IntPtr.Zero;
+
+            if (windowHandle == IntPtr.Zero)
+            {
+                return;
```

Comment:
if you return before you unsubscribe it will remain subscribed. Is that OK?

Reply:
Handling backdrop should be done in `OnSourceInitialized` and everything is fine.
There is no need to handle loaded anywhere.
The mechanic to do it this way is based on the fact that it's not part of WPF itself.

Reply:
Fixed this.